a11y/README.md

```diff
@@ -0,0 +1,28 @@
+# SYZYGY PL - Accessibility Styleguide
+
+## HTML
+
+* Always declare language attribute -- globally on your `html` wrapper and locally on elements containing text in other languages.
+* Use HTML5 sectioning elements such as `header`, `main`, `footer` `aside`, `section` to create a comprehensive landmark structure of the website or application. Also, remember about headings hierarchy (`h1` - `h6`).
+* Use semantic HTML5 elements to create components, i.e. if component requires user interaction, mark it as `a` or `button` ([more on this topic](https://marcysutton.com/links-vs-buttons-in-modern-web-applications)).
+* Remember that all the `img` elements require `alt` attribute. Also if you do not provide text content in your links (for example you use image or icon as a link), add descriptive `aria-label` attribute to your `a` element. Helpful reource: [An alt Decision Tree](https://www.w3.org/WAI/tutorials/images/decision-tree/).
+* Use ARIA attributes with understanding, remembering that [no ARIA is better than bad ARIA](https://www.w3.org/TR/wai-aria-practices/#no_aria_better_bad_aria).
+
+## CSS
+
+* Do not remove `outline: none` on focusable elements if you do not provide alternative styling.
+
+## Testing 
+
+* Test color contrast and for different types of color blindness. If some parts of application's UI do not have sufficient color contrast or are uncomprehensive for people who do not read colors, do not hesitate to discuss it with the designer you collaborate with.
```

Comment:
[CCA for mac](https://github.com/ThePacielloGroup/CCA-OSX)

Reply:
Dodaję do listy narzędzi!